recipes/aerobulk-python/meta.yaml

```diff
@@ -0,0 +1,62 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
Ping! BTW, you can probably create recipes faster and almost merge-ready with grayskull. See the comment below.

Reply:
Ah yes, thanks. I meant to do that, actually. Let me remove the hints now.